pkg/agents/team6/strategies.go

```diff
@@ -1,24 +1,93 @@
 package team6
 
+import (
+	"math"
+	"math/rand"
+
+	"github.com/SOMAS2021/SOMAS2021/pkg/utils/globalTypes/health"
+)
+
 type thresholdData struct {
-	satisficeThresh float64
-	satisfyThresh   float64
-	maxIntake       float64
+	maxIntake float64
 }
 
+type levelsData struct {
+	strongLevel  float64
+	healthyLevel float64
+	weakLevel    float64
+	critLevel    float64
+}
+
+var daysInCritical int = 0
+var foodTakeDay int
+
 func (a *CustomAgent6) foodIntake() float64 {
-	thresholds := thresholdData{satisficeThresh: 20.0, satisfyThresh: 60.0, maxIntake: 80.0}
+
+	healthInfo := a.HealthInfo()
+
+	thresholds := thresholdData{
+		maxIntake: 80.0,
+	}
+
+	levels := levelsData{
+		strongLevel:  0.6 * float64(healthInfo.MaxHP),
+		healthyLevel: 0.3 * float64(healthInfo.MaxHP),
+		weakLevel:    0.1 * float64(healthInfo.MaxHP),
+		critLevel:    0.0,
+	}
```

Comment:
Should this be an int?

Reply:
Yes, but would require a huge amount of casting both in struct and when accessed in functions